Decisions.Financial.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29926.136
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Decisions.Financial", "DecisionsFinancialCustom\Decisions.Financial.csproj", "{C1588AE9-5ECE-4885-B5C5-CA2E653093ED}"
```

Comment:
What was in this Project?

Reply:
The path was corrected. No changes to the project.